aiohttp/web_runner.py

```diff
@@ -148,12 +148,18 @@ def __init__(self, *, handle_signals=False, **kwargs):
         self._handle_signals = handle_signals
         self._kwargs = kwargs
         self._server = None
-        self._sites = set()
+        self._sites = []
```

Comment:
@asvetlov what is the motivation behind this?

Reply:
To keep a stable order of returned addresses.
Registering a new site in a runner is not a bottleneck, we can live with O(n) for it.